compiler/fbs/Converter.hx

```diff
@@ -154,7 +151,7 @@ class Converter {
 			return structObj.fields.map(function(field:FbsStructField) {
 				var fieldType:FieldType;
 				switch field.type {
-					case TPrimitive(t): 
+					case TPrimitive(_):
```

Comment:
Haxe4 language server complains about unused variable (for `t`).  Replacing this variable with `_` will remove the warning.